tests/integration/test_priority1_features.py

```
@@ -466,33 +490,37 @@ def test_text_summarization_unsupported_model(self, mock_summarizer):
 class TestWebSocketAuthentication:
     """Test WebSocket authentication and real-time processing."""
-    
+
     def test_websocket_authentication_required(self):
         """Test that WebSocket requires authentication."""
-        # This would require a WebSocket client test
-        # For now, we'll test the authentication logic
-        pass
-    
+        # Test that WebSocket endpoint requires authentication
+        with pytest.raises(Exception):  # WebSocket connection should fail without auth
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)